scripts/travis/cleanup.js

```diff
@@ -42,13 +47,60 @@ const remove = docs => {
   }
 };
 
-process.on('unhandledRejection', error => {
-  console.error('unhandledRejection', error);
-});
+const getEndDate = () => {
+  const d = new Date();
+  d.setDate(d.getDate() - DAYS_TO_KEEP);
+  return d.toISOString();
+};
+
+const getBranchBuilds = db => {
+  console.log('Querying for old branches...');
+  return db.query('builds/releases', {
+    startkey: [ 'branch', 'medic', 'medic' ],
+    endkey: [ 'branch', 'medic', 'medic', getEndDate() ],
+    limit: MAX_BUILDS_TO_DELETE
+  });
+};
+
+const getBetaBuilds = db => {
+  console.log('Querying for old beta releases...');
+  return db.query('builds/releases', {
+    startkey: [ 'beta', 'medic', 'medic', 10000 ],
+    endkey: [ 'beta', 'medic', 'medic', 0 ],
+    limit: MAX_BUILDS_TO_DELETE,
+    descending: true,
+    skip: 5 // leave the last 5 beta releases
+  });
+};
+
+const getRevs = (db, response) => {
```

Comment:
Consider rethinking the use of the term revs here: I thought you meant `getRevs` in the sense of all the revs for a document (as opposed to pulling stubs by id to get rev information for a collection of records), because that's what it would mean from CouchDB's perspective.

Reply:
Good call. Fixed.